### ISSUE TITLE
Column resize should provide conversion for width style on individual cell

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#12426]
## 📝 Provide a description of the improvement

Currently if you copy (to your clipboard) a table like:

```html
<table style="width:399px">
	<tbody>
		<tr>
			<td style="width:94px">&nbsp;</td>
			<td style="width:291px">&nbsp;</td>
		</tr>
		<tr>
			<td style="width:94px">&nbsp;</td>
			<td style="width:291px">&nbsp;</td>
		</tr>
	</tbody>
</table>
```

![Table in CKEditor 4](https://user-images.githubusercontent.com/5353898/188818745-1a1bacac-ea44-4312-9db0-4f562b926ee3.png)

And paste it into the CKEditor 5 [column resize sample](https://ckeditor.com/docs/ckeditor5/35.1.0/features/table.html#table-column-resize) it will ignore the cell widths and look in a following way:

![](https://user-images.githubusercontent.com/5353898/188820860-761cbf12-458a-4854-b990-4aaa3bae53ac.png)

The reason for this is that we didn't include _cell properties plugin_ in this sample. This plugin brings [conversion and command for cell width](https://zoom.us/j/89758913109?pwd=eGlWZFZ6R1BWdEd0WlBJN3pGSnNKQT09).

The problem is that it includes also plenty of other features (like cell background, border) that are not desired in case someone enables the column resize plugin. So we need to extract cell width to an atomic plugin that will be required by both `CellPropertiesEditing` and `TableColumnResizeEditing`.

To sum up:

1.  Let's extract cell width to `CellWidthEditing` plugin.
2.  This plugin needs to go to `requires` list of cell properties editing and column resize editing plugin.

## 📃 Other details

*   Browser: Any
*   OS: Any
*   CKEditor version: …
*   Installed CKEditor plugins: …
